Card/index.ts

```diff
@@ -57,7 +57,7 @@ export namespace Card {
 	export type Token = string
 	export namespace Token {
 		export const pattern =
-			/^(\d{10,12})\/(\d{2})\/([0-1]\d{3})\/(([0-2]\d[0-6][02468])|(((([02468][048])|([13579][2,6]))(02((0[1-9])|([1-2]\d))))|(\d{2}((((0[13578])|(1[02]))((0[1-9])|([1-2]\d)|(3[01])))|(02((0[1-9])|(1\d)|(2[0-8])))|(((0[469])|(11))((0[1-9])|([1-2]\d)|(30)))))))\/[a-z,A-Z,0-9\-_]{48}\/[a-z,A-Z,0-9\-_]{22}(\/(pan|csc|expires|month|year|masked))?$/
+			/^(\d{10,12})\/(\d{2})\/([0-1]\d{3})\/(([0-2]\d[0-6][02468])|(((([02468][048])|([13579][2,6]))(02((0[1-9])|([1-2]\d))))|(\d{2}((((0[13578])|(1[02]))((0[1-9])|([1-2]\d)|(3[01])))|(02((0[1-9])|(1\d)|(2[0-8])))|(((0[469])|(11))((0[1-9])|([1-2]\d)|(30)))))))(\/[a-z,A-Z,0-9\-_]{48}|\/([a-z,A-Z,0-9\-_]{40}))\/[a-z,A-Z,0-9\-_]{22}(\/(pan|csc|expires|month|year|masked))?$/
```

Comment:
Use a range instead: `{20,60}` or something. Makes it more forgiving and shortens the expression.

Reply:
i use a range {40,48}